### ISSUE TITLE
Missing sources for some javascript files

### ISSUE THREAD

**Issue by df7cb** [OmniDB/OmniDB#894]
I'm working on getting OmniDB into Debian. To be considered free software, Debian wants source code for everything in a program. That includes the original source for bundled `*.min.js` javascript files. OmniDB is already pretty good at providing that [*], but we are not quite there yet:

```
lintian ../omnidb_2.14.0+ds-1_source.changes
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/chartist-plugin-axistitle.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/chartist.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/flatpickr.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/jquery-1.11.2.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/jquery-ui.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/jquery.dd.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/popper.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/js/tooltip.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/lib/chart/chart.min.js
E: omnidb source: source-is-missing OmniDB/OmniDB_app/static/OmniDB_app/lib/cytoscape/cytoscape.min.js
```
Would it be possible to provide the files in the next version? Thanks!

[*] which btw is a great difference from pgAdmin4, which has only pointers to the origins of gazillions of minified javascript files it is bundling. Furthermore, the build system and minifier used there is not packaged for Debian, so getting pgAdmin4 into Debian is impossible for now.

**Comment by wind39** [comment]
Hi @ChristophBerg ,

Thank you very much for this. I will include the missing files and let you know.

Best regards,

William

**Comment by wind39** [comment]
Hi @df7cb ,

This fix reached `master` branch now that we released 2.15.

Please let us know if you need any further assistance. Closing this ticket now.

Thanks and regards,

William
